plugins/techdocs/src/alpha.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  createPlugin,
+  createSchemaFromZod,
+} from '@backstage/frontend-plugin-api';
+import { createSearchResultListItemExtension } from '@backstage/plugin-search-react/alpha';
+
+/** @alpha */
+export const TechDocsSearchResultListItemExtension =
+  createSearchResultListItemExtension({
+    id: 'techdocs',
+    at: 'plugin.search.page/items',
```

Comment:
I just realized, this should probably be the default

Reply:
Oh that's a good idea, I'll do that 🙌🏻

Reply:
Added here 🙏🏻 
https://github.com/backstage/backstage/pull/20265